src/resources/CaseAssistConfigs/CaseAssistPreviewInterfaces.ts

```diff
@@ -0,0 +1,39 @@
+import {CaseAssistConfigModel} from './CaseAssistConfigInterfaces';
+
+export interface Document {
+    title: string;
+    clickUri: string;
+    excerpt: string;
+    fields: any;
```

Comment:
It would be nice to have a stricter type here.

Reply:
I agree. The main reason is the routes don't have a swagger yet and @jeremierobert-coveo is on vacation 😅. So I don't really know what the value will look like so I was hoping someone would know